data/geoscience/geochronology/index.html

```diff
@@ -66,12 +102,50 @@ <h5>Related Resources</h5>
     info=page.Argon
   %}
 </div>
+<div class="grid package">
+  <div class="grid__col grid__col--12-of-12">
+    <h3 id="Cosmogenic">Cosmogenic</h3>
+  </div>
+  <div class="grid__col grid__col--12-of-12 package-content">
+    <img class="inline-text-left outline" src="{% link images/radio.png %}" alt="Cosmogenic" />
+    <p>Cosmogenic: Cosmogenic nuclide dating refers to a dating method that examines the interaction between cosmic rays and nuclides to provide an estimate of the age of a landform or rock surface. The Utah Geochronology Cosmogenic feature layer represents the cosmogenic dating data in the Utah Geochronology Database.</p>
+    <h5>Related Resources</h5>
+    <ul class="dotless">
+      <li><a href="https://geology.utah.gov/apps/geochron/">Utah Geochronology Database</a>
+    </ul>
+    <p></p>
+    <p>{% include contact.html subject=page.title contact=site.data.contacts.geochronology %}</p>
+  </div>
+  {% include packagedata.html
+    name="Cosmogenic"
+    info=page.Cosmogenic
+  %}
+</div>
+<div class="grid package">
+  <div class="grid__col grid__col--12-of-12">
+    <h3 id="FissionTrack">Fission Track</h3>
+  </div>
+  <div class="grid__col grid__col--12-of-12 package-content">
+    <img class="inline-text-left outline" src="{% link images/lum.png %}" alt="FissionTrack" />
+    <p>Fission Track: Fission Track dating is a radiometric dating technique that analyizes the damage of tracks/trails left by fission fragments from the decay of uranium-238. The Utah Geochronology Fission Track feature layer represents the fission track dating data in the Utah Geochronology Database.</p>
+    <h5>Related Resources</h5>
+    <ul class="dotless">
+      <li><a href="https://geology.utah.gov/apps/geochron/">Utah Geochronology Database</a>
+    </ul>
+    <p></p>
+    <p>{% include contact.html subject=page.title contact=site.data.contacts.geochronology %}</p>
+  </div>
+  {% include packagedata.html
+    name="FissionTrack"
```

Comment:
**chore**: this needs to match the front matter label. This is using the correct naming convention.

Reply:
done